apps/stable_diffusion/src/utils/utils.py

```diff
@@ -106,7 +106,7 @@ def compile_through_fx(
     model,
     inputs,
     extended_model_name,
-    is_f16=False,
+    precision="fp32",
```

Comment:
This wasn't needed.

The function already has `precision` argument!

We shouldn't change StableDiffusion's API anyway!

Reply:
Good catch, removed. It's for making `import_with_fx` more general.